lxd/tokens.go

```diff
@@ -40,6 +63,20 @@ func autoRemoveExpiredTokens(ctx context.Context, s *state.State) {
 			}
 		}
 
+		err := s.DB.Cluster.Transaction(s.ShutdownCtx, func(ctx context.Context, tx *db.ClusterTx) error {
+			for _, expiredPendingTLSIdentity := range expiredPendingTLSIdentities {
+				err := cluster.DeleteIdentity(ctx, tx.Tx(), api.AuthenticationMethodTLS, expiredPendingTLSIdentity.Identifier)
+				if err != nil {
+					logger.Warn("Failed removing pending TLS identity", logger.Ctx{"err": err, "id": op.ID()})
+				}
+			}
+
+			return nil
+		})
+		if err != nil {
+			logger.Warn("Failed removing pending TLS identities", logger.Ctx{"err": err, "id": op.ID()})
```

Comment:
can we change "id" in this line and the one above (and any others you come across) to "operation", to match that of `ImageDownload()` as I feel `id` is a bit vague here.